source/access/getting_access.rst

```diff
@@ -1,57 +1,213 @@
-to submit a project application (or you should be covered by a project
-application within your research group). For some more specialised
-hardware you have to ask access separately, typically to the coordinator
-of your institution, because we want to be sure that that (usually
-rather expensive hardware) is used efficiently for the type of
-applications for which it was purchased.
-
-Who can get a VSC account?
---------------------------
-
-Researchers at the Flemish university associations
-   In many cases, this is done through a fully automated application process,
-   but in some cases you must submit a request to your local support
-   team. Specific details about these procedures can be found on the
-   ":ref:`account request`" page in the user documentation.
-Master students in the framework of their master thesis
-   If supercomputing is needed for the thesis. For this, you will first
-   need the approval of your supervisor. The details about the procedure
-   can again be found on the ":ref:`account request`" page in the user
-   documentation.
-Use in courses at the University of Leuven and Hasselt University
-   Lecturers can also use the local Tier-2 infrastructure in the
-   context of some courses (when the software cannot run in the PC
-   classes or the computers in those classes are not powerful enough).
-   Again, you can find all the details about the application process on
-   the ":ref:`account request`" page in the user
-   documentation. It is important that the application is submitted on
-   time, at least two weeks before the start of the computer sessions.
-Researchers from iMinds and VIB
-   The application is made through
-   your host university. The same applies to researchers at the
-   university hospitals and research institutes under the direction or
-   supervision of a university or a university college, such as the
-   special university institutes mentioned in Article 169quater of the
-   Decree of 12 June 1991 concerning universities in the Flemish
-   Community.
-Researchers at other Flemish public research institutions
-   You can get compute on the Tier-1 infrastructure through a project
-   application or access the Tier-2 infrastructure through contact with
-   one of the coordinators.
-Businesses, non-Flemish public knowledge institutions and not-for-profit organisations
-   These entities can buy compute time on the
-   infrastructure. The procedures are explained on the page
-   ":ref`buying compute time`".
+also called a VSC account.  Check the `VSC website <eligible users_>`_
+for conditions.
+
+All VSC-accounts start with the letters \\"vsc\" followed by a
+five-digit number. The first digit gives information about your home
+institution. There is no relationship with your name, nor is the
+information about the link between VSC-accounts and your name publicly
+accessible.
+
+Your VSC account gives you access to most the VSC Tier-2 infrastructure, though
```

Comment:
access to most __of__ the VSC Tier-2 infrastructure

Reply:
It is not clear to me why that phrase was there?  What is not accessible by default?

Reply:
good question.. at VUB there is no hardware that you need to ask access for.